internal/services/azure_offering_template_service.go

```diff
@@ -0,0 +1,49 @@
+package services
+
+import (
+	"net/http"
+	"text/template"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+)
+
+var LighthouseTemplateFilePath = "./docs/lighthouse_template.json"
+
+type azureOfferingParams struct {
+	// OfferingDefaultName that Customer can change while deploying the offering
+	OfferingDefaultName string
+
+	// OfferingDefaultDescription describing the offering, can be changed by Customer while deploying
+	OfferingDefaultDescription string
+
+	// TenantID of the offering tenant (Azure account)
+	TenantID string
+
+	// EnterpriseAppID of the App that will act as an offering Principal
+	EnterpriseAppID string
+
+	// EnterpriseAppName of the offering principal - the display name
+	EnterpriseAppName string
+}
+
+func AzureOfferingTemplate(w http.ResponseWriter, r *http.Request) {
+	params := azureOfferingParams{
+		OfferingDefaultName:        "Red Hat Hybrid cloud console",
```

Comment:
I think product names should be capitalized (just my guts):

```suggestion
		OfferingDefaultName:        "Red Hat Hybrid Cloud Console",
```